docs/owo/ui/components/slider.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Slider
+project: owo
+---
+
+# Slider
+
+:material-xml: `#!xml <slider>`<br>
+:material-language-java: `#!java Components.slider(...)`
+
+The slider component represents a draggable slider for selecting a value within a range.
+
+**Parameters:**
+
+- `text`: The text to display alongside the slider.
+- `value`: The current value of the slider.
+- `active`: Whether the slider is active or not.
+
+**Example (Code-driven):**
+
+```java
+Components.slider(0.5)
+    .text(Text.literal("Volume"))
+    .active(true)
```

Comment:
Slider component has changed a bit. It now takes in a sizing instead of a fixed number, and the text is replaced by `message`, which takes a String -> Text function.
Example:
```java
Components.slider(Sizing.fill(50))
    .message(s -> Text.literal("Volume"))
    .active(true)
```